content/applications/hr/time_off/allocations.rst

```diff
@@ -0,0 +1,124 @@
+  determines who receives the time off allocation. The options are :guilabel:`By Employee`,
+  :guilabel:`By Company`, :guilabel:`By Department`, or :guilabel:`By Employee Tag`.
+
+  Depending on what is selected for the :guilabel:`Mode`, this following field is labeled either:
+  :guilabel:`Employees`, :guilabel:`Company`, :guilabel:`Department`, or :guilabel:`Employee Tag`.
+
+  Using the drop-down menu, indicate the specific employees, company, department, or employee tags
+  who are receiving this time off.
+
+  Multiple selections can be made for either :guilabel:`Employees` or :guilabel:`Employee Tag`.
+
+  Only one selection can be made for the :guilabel:`Company` or :guilabel:`Department`.
+- :guilabel:`Add a reason...`: if any description or note is necessary to explain the time off
+  allocation, enter it in this field at the bottom of the form.
+
+.. image:: allocations/new-allocation.png
+   :align: center
+   :alt: A new allocation form with all the fields filled out for the annual two week vacation
+         granted to all employees.
+
+.. _time_off/request-allocation:
+
+Request allocation
+==================
+
+If an employee has used all their time off, or is going to run out of time off, they can request an
+allocation for additional time. Allocations can be requested in one of two ways, either from the
+:ref:`Dashboard <time_off/dashboard>` or the :ref:`My Allocations <time_off/my-allocations>` view.
+
+To create a new allocation request, click either the :guilabel:`New Allocation Request` button on
+the main *Time Off* dashboard, or the :guilabel:`New` button in the :guilabel:`My Allocations` list
+view. Both buttons open a new allocation request form.
+
+.. note::
+   Both :guilabel:`New` buttons open a :guilabel:`New Allocation` request form, but when requested
+   from the :guilabel:`Dashboard`, the form appears in a modal. When requested from the
+   :guilabel:`My Allocations` list view, the screen navigates to a new allocation request page,
+   instead.
+
+Enter the following information on the :guilabel:`New Allocation` request form:
+
+- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+  drop-down menu. After a selection is made, the title updates with the time off type.
+- :guilabel:`Validity Period`: this field **only** appears if requesting an allocation with a time
+  off type that has the :guilabel:`Take Time Off in` field :ref:`configured
+  <time_off/time-off-officer>` to either :guilabel:`Day` or :guilabel:`Half Day`. This field does
+  **not** appear if the :guilabel:`Take Time Off in` field is set to :guilabel:`Hours`.
+
+  Using the calendar selector, select a start and end date the allocation is valid for. By default,
+  the current date populates the first field, and :guilabel:`No Limit` populates the second field.
+
+  Adjust both of these dates, if desired.
```

Comment:
I'm actually not seeing this field, even when I have the Time Off Type field set to one with a 'Take Time Off in' field set to 'Day'...
![Screenshot 2024-07-09 110026](https://github.com/odoo/documentation/assets/73958186/aef6c283-15cd-4716-87ef-d95165437d6a)
![Screenshot 2024-07-09 110037](https://github.com/odoo/documentation/assets/73958186/273df726-dbaa-4835-8eca-fdbca6b71adf)

Reply:
Please double-check this element during your revisions to ensure the accuracy of this statement

Reply:
I am checking with the experts on this, because that information was FROM the experts originally =D 
Also, in several different runbots, I keep seeing the field in ALL instances of the 'take time off in'. I am waiting to hear back, and will adjust this accordingly. Thanks for catching this!